docs/quickstarts/insights-inventory-workspace/insights-inventory-workspace.yaml

```diff
@@ -44,43 +44,43 @@ spec:
         success: Shows a success message in the task header
         failed: Shows a failed message in the task header
 
-    - title: Add systems to a group
+    - title: Add systems to a workspace
       description: |-
 
-        There are two ways to add systems to a group; you can either do it from the Group's page or from the main Systems page.
+        There are two ways to add systems to a workspace; you can either do it from the Workspace's page or from the main Systems page.
```

Comment:
should not be workspace's with apostrophe - should be "you can either do it from the Workspaces page or..."